tophatmonocle/yvm
# Use current date/time in release

## Description

- See PR title


## DevQA

### DevQA Prep
- Have a token with `repo` scope

### DevQA Steps
- Run `GITHUB_TOKEN=<token> make build_and_deploy`
- Check that a release with the current date/time was created
- Check that the release text is the commit message of the pointed-to tag
